src/lang/en/seat.php

```diff
@@ -329,6 +329,7 @@
     'allow_sso'                    => 'Allow Single Signon',
     'allow_registration'           => 'Allow Registration',
     'module_versions'              => 'SeAT Module Versions',
+    'tp_versions'                  => 'Third Party Versions',
```

Comment:
I've add à version panel below Seat version panel where puting plugins and other third party related things.

Assuming SDE is third party